app/_locales/en/messages.json

```diff
@@ -1462,6 +1462,9 @@
   "done": {
     "message": "Done"
   },
+  "dontEnableEnhancedProtection": {
```

Comment:
This key should also be added in the other localization files apart from English, I guess?

Reply:
not until we have translations